en/editor/project/index.md

```diff
@@ -81,6 +169,66 @@ If you want to replace all the preset options currently used for image asset, yo
 
 ![Layers-camera](./index/layers-camera.png)
 
+## Physics
+
+![Physics](./index/physics-index.png)
+
+This physics configuration will be in effect only when the **Physics Module** is enabled in the **Engine Module**.
+
+![Physics-In-Engine](./index/physics-in-engine.png)
+
+The physical configuration will be used in the project when preview and release. The physical effects are reflected in the control of *gravity, friction, kinetic energy transfer, and collision detection*.
```

Comment:
The physical configuration will take effect for both preview and publish environments. It includes parameters like gravity, friction, kinetic energy transfer, and collision detection